v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/dialectadapter/mysql/MysqlDialectAdapter.java

```diff
@@ -452,6 +453,16 @@ private ImmutableList<SourceColumnIndexInfo> getTableIndexes(
           stringMaxLength = null;
         }
 
+        BigDecimal decimalStepSize = null;
+        if (indexType.equals(IndexType.FLOAT) || indexType.equals(IndexType.DECIMAL)) {
```

Comment:
I am not sure if we have `FLOAT` type checked in yet.
Or should I hold on this reivew till `FLOAT` is merged in

Reply:
This PR has some overlap with the `float` PR.  It would likely be simpler to start with the `float` PR, and then we will rebase this PR on top of that one.